libs/etterlatte-ktor/src/test/kotlin/no/nav/etterlatte/libs/ktorobo/AzureAdClientTest.kt

```diff
@@ -197,17 +197,9 @@ internal class AzureAdClientTest {
                 coEvery { it.getAccessTokenForResource(any()) } returns Ok(mockk())
             }
 
-        fun genererClaimSetSystembruker() =
-            JwtTokenClaims(
-                JWTClaimsSet
-                    .Builder()
-                    .claim(Claims.idtyp.name, "app")
-                    .claim(Claims.azp_name.name, "cluster:appname:dev")
-                    .build(),
-            )
         runBlocking {
             client.hentTokenFraAD(
-                BrukerTokenInfo.of(accessToken = "a", saksbehandler = null, claims = genererClaimSetSystembruker(), idtyp = "app"),
+                systembruker(claims = mapOf(Claims.idtyp.name to "app", Claims.azp_name.name to "cluster:appname:dev")),
```

Comment:
Lurer på om vi kunne hatt en tuple som genererte `Claims.idtyp.name to "app"` for systembruker feks?

Reply:
Ja, denne ønskjer vi jo å alltid ha for systembrukar, godt poeng.